### PR TITLE
register-user

### PR DESCRIPTION
**Pull request by bugs-2209**
## Related Tickets:
- [#42178](https://edu-redmine.sun-asterisk.vn/issues/42178)
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
- This pr is to do user registration function

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by bugs-2209** [comment]
ready

File: resources/views/layouts/client/auth/register.blade.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                </div>
+                <!-- #post-## -->
+            </main>
+            <!-- #main -->
+        </div>
+        <!-- #primary -->
+    </div>
+    <!-- .col-full -->
+</div>
+@endsection
```
Comment (ghost) [inline resources/views/layouts/client/auth/register.blade.php]: eof

**Comment by bugs-2209** [comment]
ready

File: app/Http/Controllers/Client/ClientController.php [hunk 2]
```diff
@@ -25,4 +28,21 @@ public function index()
             'latestProducts'
         ));
     }
+
+    public function register()
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Client/ClientController.php]:
```suggestion
    public function getFormRegister()
```

File: app/Http/Controllers/Client/ClientController.php [hunk 3]
```diff
@@ -25,4 +28,21 @@ public function index()
...
+    {
+        return view('layouts.client.auth.register');
+    }
+
+    public function postRegister(RegisterRequest $request)
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Client/ClientController.php]:
```suggestion
    public function register(RegisterRequest $request)
```
em đặt lại tên function nhé.

**Comment by bugs-2209** [comment]
ready

**Comment by bugs-2209** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**